workos/resources/audit_logs.py

```diff
@@ -0,0 +1,52 @@
+from typing import List, Literal, Optional, TypedDict
+from typing_extensions import NotRequired
+
+from workos.resources.workos_model import WorkOSModel
+
+AuditLogExportState = Literal["error", "pending", "ready"]
+
+
+class AuditLogExport(WorkOSModel):
+    """Representation of a WorkOS audit logs export."""
+
+    object: Literal["audit_log_export"]
+    id: str
+    created_at: str
+    updated_at: str
+    state: AuditLogExportState
+    url: Optional[str] = None
+
+
+class AuditLogEventActor(TypedDict):
+    """Describes the entity that generated the event."""
+
+    id: str
+    metadata: NotRequired[dict]
+    name: NotRequired[str]
+    type: str
+
+
+class AuditLogEventTarget(TypedDict):
+    """Describes the entity that was targeted by the event."""
+
+    id: str
+    metadata: NotRequired[dict]
```

Comment:
Oh! We can use `NotRequired` instead of `total=False`.

... but [is it available in 3.8](https://peps.python.org/pep-0655/)?

Reply:
Build is green. So that answers that.

Reply:
Yep, but looks like for 3.8 -> 3.10 we have to fetch it from `typing_extensions`.